library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -78,15 +88,27 @@ internal class TransactionPayloadFragment :
     override fun onDestroyView() {
         super.onDestroyView()
         uiLoaderTask?.cancel(true)
+        fileSaverTask?.cancel(true)
     }
 
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
-        if ((type == TYPE_RESPONSE || type == TYPE_REQUEST) && body.text.isNotEmpty()) {
-            val searchMenuItem = menu.findItem(R.id.search)
-            searchMenuItem.isVisible = true
-            val searchView = searchMenuItem.actionView as SearchView
-            searchView.setOnQueryTextListener(this)
-            searchView.setIconifiedByDefault(true)
+        if ((type == TYPE_RESPONSE || type == TYPE_REQUEST)) {
+            if (body.text.isNotEmpty()) {
+                val searchMenuItem = menu.findItem(R.id.search)
+                searchMenuItem.isVisible = true
+                val searchView = searchMenuItem.actionView as SearchView
+                searchView.setOnQueryTextListener(this)
+                searchView.setIconifiedByDefault(true)
+            }
+
+            val saveMenuItem = menu.findItem(R.id.save_body)
+
+            // SAF is not available on pre-Kit Kat so let's hide the icon.
+            saveMenuItem.isVisible = Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT
```

Comment:
I would suggest adding braces for logical conditions in such cases to make it easier to read.